src/modules/setup.ts

```diff
@@ -39,22 +39,31 @@ class ReplaceVariablesStream extends Transform {
     // eslint-disable-next-line @typescript-eslint/naming-convention
     _transform(chunk: any, _: string, callback: TransformCallback) {
         let result = this._variables.substitute(chunk.toString());
+        let noticeComment: string;
         const notice = 'This file is maintained by velocitas CLI, do not modify manually. Change settings in .velocitas.json';
         const shebang = '#!/bin/bash';
+        const tplStart = '<?xml version="1.0" ?>';
```

Comment:
This is called an XML declaration. Rename constant to `xmlDeclaration` ? Probably you want this to be a regular expression to find special cases, i.e. with multiple whitespaces.